example/ToT/methods/bfs.py

```
@@ -0,0 +1,110 @@
+    if to_print:
+        print(ys)
+    return ys, {'steps': infos}
+
+
+def naive_solve(args, task, idx, to_print=True):
+    global gpt
+    gpt = partial(gpt, model=args.backend, temperature=args.temperature)
+    print(gpt)
+    x = task.get_input(idx)  # input
+    ys = get_samples(task, x, '', args.n_generate_sample, args.prompt_sample, stop=None)
+    return ys, {}
```

Ruff

No newline at end of file

Add trailing newline

(W292)